content/software-security/learning-labs/_index.md

```diff
@@ -9,6 +9,7 @@ images: []
 tags: ["Learning Labs", "Overview"]
 toc: true
 hidepageslist: true
+contentType: "conceptual"
```

Comment:
I think all the learning labs types could use more thought, most are more hands on than conceptual, no?